flexxamples/howtos/openlayers.py

```diff
@@ -0,0 +1,177 @@
+from flexx import flx
+import os
+
+BASE_DIR = os.getcwd()
+
+with open(BASE_DIR + '/static/js/ol-v4.6.4/ol.css') as f:
+    ol_css = f.read()
+
+with open(BASE_DIR + '/static/js/ol-v4.6.4/ol.js') as f:
+    ol_js = f.read()
+
+flx.assets.associate_asset(__name__, 'ol.css', ol_css)
+flx.assets.associate_asset(__name__, 'ol.js', ol_js)
+
+geojson = '{"type":"FeatureCollection","features":[{"type":"Feature","geometry":{"type":"Point","coordinates":[-82.08160400390625,23.0548095703125]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-81.31256103515625,22.510986328125]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-80.47210693359375,22.6483154296875]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-80.16448974609375,22.3187255859375]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-81.00494384765625,22.2857666015625]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-81.54876708984375,22.3406982421875]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-82.56500244140625,23.0767822265625]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-82.40570068359375,22.9449462890625]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-82.33428955078125,23.1756591796875]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-82.30682373046875,23.082275390625]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-82.21343994140625,22.9669189453125]}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-82.10357666015625,23.192138671875]}}]}'
```

Comment:
This line is too long. This either needs to be written as a multiline string, or perhaps add a ` # noqa` to suppress this style error.